front/app/containers/App/index.tsx

```diff
@@ -500,14 +497,13 @@ class App extends PureComponent<Props, State> {
                     </Suspense>
                   </ErrorBoundary>
                   <ErrorBoundary>
-                    <LoadableModal
-                      opened={userDeletedSuccessfullyModalOpened}
-                      close={this.closeUserDeletedModal}
-                    >
-                      <LoadableUserDeleted
+                    <Suspense fallback={null}>
+                      <UserDeletedModal
+                        modalOpened={userDeletedSuccessfullyModalOpened}
+                        closeUserDeletedModal={this.closeUserDeletedModal}
                         userSuccessfullyDeleted={userSuccessfullyDeleted}
                       />
-                    </LoadableModal>
+                    </Suspense>
```

Comment:
Not related to routes, but replaced this because LoadableUserDeleted was still using LoadableLoadingCitizen, which was its last instance.